### PR TITLE
[RW-10520][risk=no] Updated identity providers to use a single data access modules.

### PR DESCRIPTION
**Pull request by evrii**
---
**PR checklist**

- [ ] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

File: api/db/changelog/db.changelog-221-update-access-modules.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      WHERE access_module_id = (
+        SELECT access_module_id
+        FROM access_module
+        WHERE name = 'RAS_ID_ME')
+    </sql>
+    <sql>
+      DELETE FROM access_module
+      WHERE name = 'RAS_ID_ME'
+    </sql>
+    <sql>
```
Comment (evrii) [inline api/db/changelog/db.changelog-221-update-access-modules.xml:38]: Dropping ID.me and Login.gov

File: api/src/main/java/org/pmiops/workbench/ras/RasLinkService.java [hunk 2]
```diff
@@ -154,10 +152,9 @@ public DbUser linkRasAccount(String authCode, String redirectUrl) {
 
     String username = getUsername(userInfoResponse);
     DbUser user;
-    if (username.toLowerCase().contains(ID_ME_IDENTIFIER_LOWER_CASE)) {
-      user = userService.updateRasLinkIdMeStatus(username);
-    } else if (username.toLowerCase().contains(LOGIN_GOV_IDENTIFIER_LOWER_CASE)) {
-      user = userService.updateRasLinkLoginGovStatus(username);
+    if (username.toLowerCase().contains(ID_ME_IDENTIFIER_LOWER_CASE)
```
Comment (evrii) [inline api/src/main/java/org/pmiops/workbench/ras/RasLinkService.java:155]: Consolidating for now, but will probably end up splitting up again when I start track identity verifications in its own table.

**Comment by Peter-Lavigne** [comment]
Why was the migration "unnecessary"?

Also, it seems like there's some unrelated changes showing up in the files changed. Specifically, I'm seeing changes to the `aou-utils` and `tanagra` submodules. I'm not familiar with submodules, any idea why that's happening?

**Comment by Peter-Lavigne** [comment]
> Why was the migration "unnecessary"?

Never mind, I had this PR confused with https://github.com/all-of-us/workbench/pull/7823

**Pull request closed without merging**